contrib/tools/psbtsigner/testing_keys.sh

```diff
@@ -0,0 +1,3 @@
+S1KEY="xprv9whdJhXXBMAUfFPpeJkSfyCjdGDcs3FiWKL6Xn4hKgpYQ6uaCnTkZhND8Cy4NwtAVhazDrcWgqLJGWc7mLt7iG2hvVKZg4sPf8pavLFJRUH"
+S2KEY="xprv9xDe7xTWoLhKtu6BcLuWqwNVUTEE6v8j5jahb75rTsQD1RctZLVHLKXNdr7snbgt7BH9vRvCNx9xdCQnSvGnwZyAZmFwTuKuDyU8S3xVvnz"
+M1KEY="xprv9wC5JsVNK9HwGcCAuZGZE5ijdV2ST232JSXvCaT7jfWrGHqZn9EJGW76XNFtxbH7TwBQR1WSzhsiG2wwudDkJZi4ic4HJfDUcwRwHKgYufK"
```

Comment:
![image](https://user-images.githubusercontent.com/22457751/117844103-b5770900-b27f-11eb-9a71-fc6349584491.png)

Reply:
Ahahah btw those were my testing keys that absolutely didn't have to be included in the commit 🤦🏻 Thanks god we're in regtest